cli/cmd/ai.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	cloudquery_api "github.com/cloudquery/cloudquery-api-go"
+	"github.com/cloudquery/cloudquery/cli/v6/internal/api"
+	"github.com/samber/lo"
+
```

Comment:
I know there's no `make goimports` target for the cli, but we should still remove this line and sort the imports.